### PR TITLE
Euclidean traveling salesperson

### PR DESCRIPTION
**Pull request by katelynbottcher**
https://github.com/Watts-Lab/task-mapping/issues/85

### CODE REVIEW

File: Euclidean traveling salesperson.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+![TSP](images/TSP.png)
+
+# Procedure
+## Steps
+> Obtain informed consent
+> Each subject took a nine-problem pretest, in part to train them on the TSP and in part to assess their individual abilities with respect to solving TSPs
+> Phase One:
+> > Subjects were asked to click with the computer’s mouse on the city icons in the sequence corresponding to the path they wished to submit as their answer. The computer program would draw a line segment connecting each pair of cities as soon as the second city in each leg of the journey had been clicked on.
+> > Each map consisted of 25 cities and thus required 25 separate legs of the overall journey to complete. Participants complete 6 different maps. 
+> > They had 17 tries per map, and 50 seconds per map.
```
Comment (sarikasu) [inline Euclidean traveling salesperson.md:22]: subjects could see the solution they entered in the previous round, in a smaller window below the main task window, along with its distance. 

 If the subject had not submitted an answer by the end of the 50 s, the interface automatically moved on.

File: Euclidean traveling salesperson.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+> Phase Two:
+> > Subjects were put into groups of three, and assigned to one of three conditions.
```
Comment (sarikasu) [inline Euclidean traveling salesperson.md:24]: Subjects were randomly matched with and anonymous to each other; the order of treatments was also randomly assigned and subjects were exposed to multiple treatments

File: Euclidean traveling salesperson.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Summary
+This task involves finding the shortest path among a number of “cities” on a map to mimic complex problem solving in which each aspect of a problem depends on other aspects of that problem.
+
+# References
+> Main paper: https://www.pnas.org/content/pnas/115/35/8734.full.pdf
+> > Supporting paper: https://www.pnas.org/content/pnas/suppl/2018/08/09/1802407115.DCSupplemental/pnas.1802407115.sapp.pdf 
+
```
Comment (sarikasu) [inline Euclidean traveling salesperson.md:7]: Data for replication: https://dataverse.harvard.edu/dataset.xhtmlpersistentId=doi:10.7910/DVN/TSSQLY&version=2.0

**Review by sarikasu — COMMENTED** [review]
Should I wait to approval until you hear back on instructions?

**Comment by katelynbottcher** [comment]
> Should I wait to approval until you hear back on instructions?

Yes, I'll let you know.

**Comment by katelynbottcher** [comment]
> Should I wait to approval until you hear back on instructions?

The researchers don't have instructions.

**Review by sarikasu — APPROVED** [review]

**Pull request merged**